sklearn/_build_utils/deprecated_modules.py

```diff
@@ -0,0 +1,37 @@
+"""Generates submodule to allow deprecation of submodules and keeping git
+blame."""
+from pathlib import Path
+
+
+_DEPRECATED_MODULES = {
+    # TODO: Remove in 0.24
```

Comment:
Please add comment indicating that the 3tuple is (underscored_filed, deprecated_path, correct_path)